app/templates/administrative-units/new.hbs

```diff
@@ -0,0 +1,204 @@
+              <:label>Type bestuur</:label>
+              <:content>
+              <ClassificationSelect @selected={{this.model.administrativeUnit.classification}}
+                  @onChange={{fn (mut this.model.administrativeUnit.classification) }} />
+              </:content>
+            </Item>
+            <Item @labelFor="denomination">
+              <:label>Strekking</:label>
+              <:content>
+                <AuInput   @value={{this.model.worshipService.denomination}} id="denomination" />
+              </:content>
+            </Item>
+          </:left>
+          <:right as |Item|>
+            <Item @labelFor="status-select">
+              <:label>Status</:label>
+              <:content>
+                  <OrganizationStatusSelect
+                    @selected={{this.model.administrativeUnit.organizationStatus}}
+                    @onChange={{fn
+                      (mut this.model.administrativeUnit.organizationStatus)
+                    }}
+                  />
+              </:content>
+            </Item>
+          </:right>
+        </Card.Columns>
+        <Card.Columns>
+          <:left as |Item|>
+            <Item>
+              <:label>Grensoverschrijvend</:label>
+              <:content>
+                <AuControlRadio @label="Ja" @name="is-cross-border" @value={{this.model.worshipService.crossBorder}}
+                  @identifier="cross-border-true" @onChange={{fn (mut this.model.worshipService.crossBorder) true }}/>
+                <br />
+                <AuControlRadio @label="Nee" @name="is-cross-border" @value={{this.model.worshipService.crossBorder}}
+                  @identifier="cross-border-false" @onChange={{fn (mut this.model.worshipService.crossBorder) false }}/>
+              </:content>
+            </Item>
+            <Item @labelFor="administrative-unit-scope">
+              <:label>Werkingsgebied</:label>
+              <:content>
+                {{! TODO: Enable multiple selections and concat the selections in one label }}
+                <ScopeSelect @selected={{this.model.administrativeUnit.scope}} @onChange={{fn (mut
+                  this.model.administrativeUnit.scope)}} />
+              </:content>
+            </Item>
+          </:left>
+          <:right as |Item|>
+            <Item>
+              <:label>KBO Nummer</:label>
+              <:content>
+                <AuInput @value={{this.model.structuredIdentifierKBO.localId}}  
+                  id="kbo-identifier" />
+              </:content>
+            </Item>
+            <Item>
+              <:label>Sharepoint Identificator</:label>
+              <:content>
+                <AuInput @value={{this.model.structuredIdentifierSharepoint.localId}}  
+                  id="sharepoint-identifier" />
+              </:content>
+            </Item>
+          </:right>
+        </Card.Columns>
+      </:card>
+    </EditCard>
+
+    <EditCard>
+      <:title>Primaire contactgegevens</:title>
```

Comment:
In the edit of the administrative units it looks similar, there isn't that kind of structure division, should I proceed?

Reply:
I think it makes sense to make them all consistent. I prefer the address left' other data right approach since that makes keyboard navigation work correctly. I'll double check with Saska.

Reply:
Maybe just keep it like this for now (unless you did it already ofc), we can change the order later.